.github/workflows/main.yml

```
@@ -74,7 +74,13 @@ jobs:
           tag_name: ${{ steps.git_tag.outputs.tag }}
           release_name: Release ${{ steps.git_tag.outputs.tag }}
           body: ${{ steps.git_tag.outputs.changes }}
-      - name: Prepare chart for publishing
+      - if: github.ref == 'refs/heads/master'
+        name: Check if the branch is master 
+        id: branch_name
+        run: |
+          echo ::set-output name=master::true
+      - if: '${{ steps.branch_name.outputs.master }}'
+        name: Prepare chart for publishing
```

```suggestion
      - if: github.ref == 'refs/heads/master'
        name: Prepare chart for publishing
```

This is the same, right? Why set the variable?

But only publishing on master seems like a good option to me, now the releases page of `otomi-core` is producing a lot of noise...